### ISSUE TITLE
[R13] Condition Help 

### ISSUE THREAD

**Issue by maskelihileci** [x64dbg/x64dbg#2377]
Debugger version = oct 24.2018, 13:00:44
 Operating system version = Windows 10 - 1607  x64
 Brief description of the issue = Reading the value of the register address

([R13] == 0x4D & RCX == 0x32 & R11 == 0x574)

I'm doing it this way it doesn't work

**Comment by mrexodia** [comment]
Likely r13 is not a valid memory location, or you expect a byte. use `ReadByte(r13) == 0x4d`. See https://help.x64dbg.com/en/latest/introduction/Expression-functions.html#byte-word-dword-qword-ptr

**Comment by rokups** [comment]
I am also having trouble to break using a memory location in a breakpoint condition.
![image](https://user-images.githubusercontent.com/19151258/108854846-fcdee900-75f0-11eb-8351-3b9939b31df2.png)
This breakpoint does log 41595, but never breaks. Documentation has all kinds of useful examples except this one, which made me wonder if this is even supported?

**Comment by mrexodia** [comment]
Expressions use hex as the default number base (which should be in the
documentation). Try “==.41595”

**Comment by rokups** [comment]
Yes that was my mistake, thanks. This is why i deleted my post once i 
realized this mistake.

-- Rokas Kupstys

**Comment by mrexodia** [comment]
Ah sorry 😄 I usually reply from my email directly so didn’t see your post
was deleted.
